scripts/fdbshow

```diff
@@ -63,25 +63,51 @@ class FdbShow(object):
             return
 
         oid_pfx = len("oid:0x")
+
+        client = self.db.redis_clients["ASIC_DB"]
+        pipe = client.pipeline()
+        values = []
+        fdb_list = []
         for s in fdb_str:
             fdb_entry = s.decode()
             fdb = json.loads(fdb_entry .split(":", 2)[-1])
             if not fdb:
                 continue
-
-            ent = self.db.get_all('ASIC_DB', s, blocking=True)
+            fdb_list.append(fdb)
+            pipe.hgetall(s)
+        values = pipe.execute()
+        client = self.db.redis_clients["ASIC_DB"]
+        pipe = client.pipeline()
+        vlans = []
+        vlan_keys = self.db.keys('ASIC_DB', "ASIC_STATE:SAI_OBJECT_TYPE_VLAN:*")
+        for v in vlan_keys:
+            pipe.hgetall(v)
+        vlans = pipe.execute()
+
+        posi = 0
+        for ent in values:
+            if 'SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID' not in ent:
+                posi = posi + 1
```

Comment:
Is this case true, where SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID is not present?

Reply:
If an entry is deleted either due to ageout or flush just after we fetched the keys, it will hit this case.